teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -39,9 +50,15 @@ public int getEth1LogsMaxBlockRange() {
   }
 
   public void configure(final TekuConfiguration.Builder builder) {
+    ArrayList<String> mergedEth1Endpoints = new ArrayList<>();
+    if (eth1Endpoint != null) mergedEth1Endpoints.add(eth1Endpoint);
+    if (eth1Endpoints != null) mergedEth1Endpoints.addAll(eth1Endpoints);
+
     builder.powchain(
         b ->
-            b.eth1Endpoint(Optional.ofNullable(eth1Endpoint))
+            b.eth1Endpoints(
+                    Optional.ofNullable(
+                        mergedEth1Endpoints.isEmpty() ? null : List.copyOf(mergedEth1Endpoints)))
```

Comment:
I *think* this should just be an empty list when there are no eth1 endpoints now rather than needing to use an `Optional`.  I might be missing some reason that doesn't work but it would definitely be nicer to just use an empty list if we can get away with it. I think just updating `PowchainConfiguration.isEnabled` is the key.

Reply:
If I remember correctly there was a test failing if using empty list. But I agree, the code simplifies using an empty list.

Reply:
I'm guessing that's a config test and it could just be updated - the point of the test is really just that the case of "no endpoints" is handled properly.

Reply:
yeah that's exactly what i'm dealing with right now.. but I have to stop for the moment. Back to main job :)
I'll finish later today.